packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -202,6 +202,15 @@ export function createKusama (t: TFunction): EndpointOption {
         providers: {
           SubGame: 'wss://gamma.subgame.org/'
         }
+      },
+      {
+        info: 'loomNetwork',
+        homepage: 'https://loomx.io/',
```

Comment:
```suggestion
        isUnreachable: true,
        homepage: 'https://loomx.io/',
```

Since the RPC is not (yet) operational.